app.js

```diff
@@ -0,0 +1,156 @@
+const addMovieBtn = document.getElementById("add-movie-btn");
+
+const movieModal = document.getElementById("add-modal");
+
+const backdrop = document.getElementById("backdrop");
+
+const cancelBtn = movieModal.querySelector(".btn--passive");
+
+const addBtn = cancelBtn.nextElementSibling;
+
+const userInputs = movieModal.querySelectorAll("input");
+const enteryText = document.getElementById("entry-text");
+
+const deleteModal = document.getElementById("delete-modal");
+
+const dangerBtn = document.getElementById("btn-danger");
+
+let successBtn = document.getElementById("btn-success");
+
+const movies = [];
+
+const toggleBackdrop = () => {
+  backdrop.classList.toggle("visible");
+};
+
+const removeBackdrop = () => {
+  clearUsrInputs();
+  closeMovieModal();
+  cancelMovieDeletion();
+};
+
+const clearUsrInputs = () => {
+  for (const usrInput of userInputs) {
+    usrInput.value = "";
+  }
+};
+
+const cancelAddMovie = () => {
+  toggleBackdrop();
+  closeMovieModal();
+  clearUsrInputs();
+};
+
+const updateUi = () => {
+  if (movies.length == "") {
```

Comment:
```suggestion
  if (!movies.length) {
```

Reply:
thankyou for your seggestion @Tanvir04khan . i made those changes , you can check my commits.